crates/api_crud/src/site/read.rs

```diff
@@ -69,6 +69,7 @@ impl PerformCrud for GetSite {
             application_question: setup.application_question.to_owned(),
             private_instance: setup.private_instance,
             default_theme: setup.default_theme.to_owned(),
+            default_post_listing_type: setup.default_post_listing_type.to_owned(),
```

Comment:
I really doubt that it makes sense to populate all of these through the config, most likely no one is using that. Besides, the values arent verified at all. It would make more sense to use config.setup only for admin username/password, and show the setup page in lemmy-ui after admin login.

Reply:
To me it would be more complicated code-wise for them not to match. If you're going to setup an admin user through the config, why not have all the options? Also just because we're not using them doesn't mean someone might not want to use them in the future, say setting a default theme or sidebar.

Reply:
Because its not clear at all which values are valid, and there is no verification either so its easy to mess up. There are countless things that someone "might want to use in the future", but if no one actually uses it, it might be completely broken in practice, and is simply a waste of time to maintain.